internal/security/grantvmgroupaccess.go

```diff
@@ -1,17 +1,17 @@
+//go:build windows
 // +build windows
 
 package security
 
 import (
+	"fmt"
 	"os"
 	"syscall"
 	"unsafe"
-
-	"github.com/pkg/errors"
 )
 
 type (
-	accessMask          uint32
+	AccessMask          uint32
```

Comment:
It feels weird that this is the only type exported, especially since that means that users can define their own.
Would it be better to define an `AccessMaskNone = 0` that you can use in `grantvmgroupaces`?
```go
desiredAccess :=  security.AccessMaskNone 
if *readFlag {
	desiredAccess |= security.AccessMaskRead
}
```
or have?
```go
func AccessMask(read, write, execute, all bool) accessMask { ... }
```

Reply:
I'd prefer former in this case... having a factory function probably means not exporting the `AccessMaskX` constants and then it becomes weird when it comes to defining default values, since we'd need to call into `AccessMask(...)` for that.